tools/importer-msgraph-metadata/internal/pipeline/task_parse_resources.go

```diff
@@ -586,5 +584,91 @@ func (p pipelineForService) parseResources(resourceIds parser.ResourceIds, model
 		//resource.Category = normalize.DeDuplicateName(resource.Category)
 	}
 
+	// Look for resources with no operations, as well as duplicate operations. For duplicate operations drop the one with the
```

Comment:
This is the important part of this change - after parsing all resources (and their operations) for a service, we iterate them and look for operations with duplicate names. For any duplicates that are discovered, we remove them according to the code comments, and log what we did. This prevents flapping diffs because the parsing order is nondeterministic, so for each import, either of the conflicting operations may "win". This has the side effect of losing support for some operations, however as per the code comments this is probably an acceptable compromise.